tests/testthat/test-basics.R

```diff
@@ -426,3 +426,37 @@ test_that("data argument is checked in recipe.formula() (#1325)", {
     recipe(~ ., data = data)
   )
 })
+
+test_that("step constructor", {
+
+  step_lightly  <-
+    function(trained = FALSE, skip = FALSE, id = "id") {
+      step(
+        subclass = "lightly",
+        trained = trained,
+        skip = skip,
+        id = id
+      )
+    }
+
+  expect_snapshot(
+    step_lightly(trained = "yes"),
```

Comment:
top-tier function name. but it doesn't reflect the call structure of a step, as this is what `step_lightly_new()` looks like.

we can instead use something like

```suggestion
    recipe(~., mtcars) %>% step_normalize(trained = "yes"),
```